### ISSUE TITLE
inset-area is deprecated

### ISSUE THREAD

**Issue by Wurstnase** [db-ui/core#983]
https://developer.chrome.com/blog/new-in-chrome-129/

```
The 'inset-area' property has been deprecated, and will be removed from this browser very soon. Please use the 'position-area' property instead.
```

https://github.com/db-ui/core/blob/f52e0d33c5e49aa79322580cf1890309545c2077/source/_patterns/01-elements/_form-elements.scss#L118

**Comment by mfranzke** [comment]
duplicate of https://github.com/db-ui/core/issues/981 (this is most likely, why @Wurstnase has closed it already).

**Comment by mfranzke** [comment]
the fix will be released shortly with release v3.2.4
